webpack.proxy.template.js

```diff
@@ -22,5 +22,8 @@ module.exports = {
   //     req.url = req.url.replace(/^\/mesos/, '');
   //   }
   // },
-  "*": "http://m1.dcos"
+  "*": {
```

Comment:
hoping we are going to keep the `webpack` folder to organize webpack modules instead having all in the root creating noise.